src/pages/page_layout.cr

```diff
@@ -18,6 +18,11 @@ abstract class PageLayout
 
       body class: "font-sans text-grey-darkest leading-tight bg-grey-lightest" do
         mount Shared::Header, @context.request
+        if responds_to?(:hero_content)
+          div class: "hero_content" do
+            render_if_defined :hero_content
+          end
+        end
```

Comment:
The default page puts all of the content inside of a `container`. This section allows us to do that gradient that spans 100%